### DISCUSSION TITLE
Welcome to flagship Discussions!

### DISCUSSION THREAD

**Discussion by videege** [videege/flagship#72] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gdf20** [comment]
Hi @videege 

I used the Flagship site during the pandemic and now that in person play is back, had been looking to start tracking a new campaign. 

However we seem to have an issue where when a user clicks on an invite link and accepts the invite, they are unable to access the campaign, only getting a "loading" page. 

Similarly when trying to access the newly invited too campaign from the campaigns page, it is not visible. 

Can you assist?
